app/src/main/java/org/wikipedia/database/contract/UserLandmarkContract.java

```diff
@@ -0,0 +1,22 @@
+package org.wikipedia.database.contract;
+
+import android.net.Uri;
+
+import org.wikipedia.database.column.IdColumn;
+import org.wikipedia.database.column.LongColumn;
+import org.wikipedia.database.column.StrColumn;
+
+/**
+ * Created by Artem on 2018-04-12.
+ */
+
+public interface UserLandmarkContract {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [interfaces should describe a type and hence have methods.](https://app.codacy.com/app/amawai/android-wikipedia-390/pullRequest?prid=1541219)